ui/src/components/VolumeListTable.js

```diff
@@ -151,6 +100,13 @@ const TooltipContent = styled.div`
   min-width: 60px;
 `;
 
+const UnknownIcon = styled.i`
```

Comment:
This component is duplicated in table row, please consider exporting it instead if it is necessary